server/plugin/webhook.go

```diff
@@ -543,6 +554,8 @@ func (p *Plugin) handlePRDescriptionMentionNotification(event *github.PullReques
 
 		p.sendRefreshEvent(userID)
 	}
+
+	p.API.LogDebug("Event Completed.", "Event", "handlePRDescriptionMentionNotification")
```

Comment:
Unsure if we need this logging. We definitely want to make sure we log any errors that occur in the whole process of processing the webhook though